source/_docs/eu-region.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Create a New Site in Pantheon's EU Region
-description: Learn how to create a new site located in the EU.
+title: Early Access: EU Region Availability
```

Comment:
"EU Region Limited Availability"

Reply:
New Product Lifecycle terms.... "Early Access" is a distinct phase that's before "Limited Availability"